actors/account/tests/account_actor_test.rs

```diff
@@ -128,7 +140,7 @@ fn authenticate_message() {
     });
     assert_eq!(
         ExitCode::USR_ILLEGAL_ARGUMENT,
-        rt.call::<AccountActor>(3, &params).unwrap_err().exit_code()
+        rt.call::<AccountActor>(3, params).unwrap_err().exit_code()
```

Comment:
Hm. We should probably continue to take this by reference. Probably `Option<&IpldBlock>`. One can always go from `&Option<IpldBlock>` to `Option<&IpldBlock>` for free.

Reply:
Although it isn't _critical_.

Reply:
(I'll trust your API sense)